test/metabase/api/card_test.clj

```diff
@@ -10,8 +10,20 @@
             [metabase.api.pivots :as pivots]
             [metabase.driver.sql-jdbc.execute :as sql-jdbc.execute]
             [metabase.http-client :as http]
-            [metabase.models :refer [Card CardFavorite Collection Dashboard Database ModerationReview
-                                     Pulse PulseCard PulseChannel PulseChannelRecipient Table ViewLog]]
+            [metabase.models
```

Comment:
this looks like it got autoformatted for no go reason

Reply:
Hmm I thought the namespace linter was telling me I should do this on a previous commit, but maybe I was imagining that